plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -106,6 +108,22 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 		fieldsRef.current?.postcode?.revalidate();
 	}, [ currentCountry ] );
 
+	// Focus the first input when opening the form.
+	useEffect( () => {
+		if ( ! isFirstRender.current && isEditing && fieldsRef.current ) {
```

Comment:
We want to skip the first render because if the user doesn't have a Shipping address saved, for example, the form will start opened and we don't want to focus the first input in this case.